src/listeners/github/message-handler.js

```diff
@@ -10,22 +11,19 @@ const githubWebhooks = new Set([
   config.get('github.repos.cdpSquidProxy'),
   config.get('github.repos.cdpGrafanaSvc')
 ])
-const validActions = new Set(['workflow_run'])
 
-const shouldProcess = (message) => {
-  const eventType = message.github_event
+const shouldProcess = (message, eventType) => {
   const repo = message.repository?.name
-  return validActions.has(eventType) && githubWebhooks.has(repo)
+  return message.github_event === eventType && githubWebhooks.has(repo)
 }
 
 const handle = async (server, message) => {
-  if (!shouldProcess(message)) {
-    return
-  }
-
-  if (message.github_event === 'workflow_run') {
+  if (shouldProcess(message, 'workflow_run')) {
     return await workflowRunHandlerV2(server, message)
```

Comment:
wondering if we should try/catch these, just in case it fails, throw an exception and we never raise the alert?